web/styles/user_status.css

```diff
@@ -31,24 +32,21 @@
                 height: 20px;
                 cursor: pointer;
                 /*
-                    the following rule is not necessarily better than the one
-                    from a6bef5154197b15c20445526fd3f219b4f2e5379 but it works.
+                the following rule is not necessarily better than the one
```

Comment:
Go ahead delete these comments and the `top: -2px`. `.selected-emoji` is no longer a positioned element, so that `top` value has no effect--much like  your changes to `.smiley-icon` below.